help/configure-home-view.md

```diff
@@ -80,5 +80,5 @@ shortcut.
 
 * [Reading strategies](/help/reading-strategies)
 * [Recent conversations](/help/recent-conversations)
-* [All messages](/help/all-messages)
+* [Global feed](/help/global-feed)
```

Comment:
If you're renaming a /help/ article, you should add a redirect to `zerver/lib/url_redirects.py`.